library/std/src/sys/pal/windows/c.rs

```diff
@@ -136,7 +112,7 @@ pub struct SYMBOLIC_LINK_REPARSE_BUFFER {
     pub PrintNameOffset: c_ushort,
     pub PrintNameLength: c_ushort,
     pub Flags: c_ulong,
-    pub PathBuffer: WCHAR,
+    pub PathBuffer: u16,
```

Comment:
So this is a type alias I think is actually worth preserving, because WCHAR indicates it should be interpreted as a UTF-16 encoded pair of bytes.

Unless in this case that's meaningless, in which case nah.

Reply:
So part of the issue with these aliases is we're very inconsistent. Without anything to enforce it we end up with a mix of aliases and not aliases. It's easier to lean into that tendency and just not do aliasing. The ultimate fix would be to have proper types for Windows strings and characters. But that's a much bigger job.

Reply:
That said, I'm not against using WCHAR on a best effort basis,

Reply:
Sure! I'm fine with this change then.

Reply:
Right, I think the reason to keep WCHAR around would be so that if we do ever get around to such a "real" change, it would be easier to do, as that would distinguish it from e.g. a random `c_ushort`.